applications/SwimmingDEMApplication/test_examples/PFEMcoupledDEMexample.gid/ProjectParametersDEM.json

```diff
@@ -101,8 +100,8 @@
 "debug_tool_cycle_comment"                       : " number of 'ticks' per debug computations cycle",
 "similarity_transformation_type"         : 0,
 "similarity_transformation_type_comment" : " no transformation (0), Tsuji (1)",
-"dem_inlet_element_type"                 : "SphericSwimmingParticle3D",
-"dem_inlet_element_type_comment"         : " SphericParticle3D, SphericSwimmingParticle3D",
+"dem_inlet_element_type"                 : "SwimmingParticle3D",
```

Comment:
These changes in the json files are also dangerous in terms of GUI...

Reply:
idem as above